operator/cmd/mesh/install.go

```diff
@@ -190,3 +209,41 @@ func savedIOPName(iop *v1alpha12.IstioOperator) string {
 	}
 	return ret
 }
+
+// DetectIstioVersionDiff will show warning if istioctl version and control plane version are different
+// nolint: interfacer
+func DetectIstioVersionDiff(cmd *cobra.Command, tag string, kubeClient kube.ExtendedClient) error {
+	icps, err := kubeClient.GetIstioVersions(context.TODO(), controller.IstioNamespace)
+	if err != nil {
+		return err
+	}
+	if len(*icps) != 0 {
+		var icpTag string
+		for _, icp := range *icps {
+			tagVer, err := GetTagVersion(icp.Info.GitTag)
+			if err != nil {
+				return err
+			}
+			if tagVer != "" && tag != tagVer {
```

Comment:
how to handle multiple control plane versions? seems like we are warning only one here?

Reply:
please see https://github.com/istio/istio/pull/24263#issuecomment-707629340